src/MudBlazor/Styles/components/_dropzone.scss

```
@@ -17,8 +17,23 @@
 }
 
 .mud-drop-item:not(.mud-drop-item-preview-start) {
+  cursor: default;
+  user-select: none;
+}
+
+.mud-drop-item:not(.mud-drop-item-preview-start):not(:has(.mud-drag-handle)) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)